flink-runtime/src/main/java/org/apache/flink/runtime/resourcemanager/slotmanager/FineGrainedSlotManager.java

```diff
@@ -710,7 +705,7 @@ private void releaseIdleTaskExecutorIfPossible(TaskManagerInfo taskManagerInfo)
 
     private void releaseIdleTaskExecutor(InstanceID timedOutTaskManagerId) {
         final FlinkException cause = new FlinkException("TaskManager exceeded the idle timeout.");
-        LOG.info(
+        LOG.debug(
```

Comment:
We could make this logging the responsibility of the resource manager. Then it will really only be printed if we are shutting a TM down.

Reply:
It is logged in `ResourceManager#closeTaskManagerConnection`, will remove it.